src/util/utils.js

```diff
@@ -613,3 +613,19 @@ export function getDenomination(currencyCode: string, settings: Object): EdgeDen
 export function getDefaultDenomination(currencyCode: string, settings: Object): EdgeDenomination {
   return settings[currencyCode].denominations.find(denomination => denomination.name === currencyCode)
 }
+
+export function checkFilterWallet(wallet: GuiWallet, filterText: string): boolean {
```

Comment:
This function is not needed, just use the function below and always pass the wallet currency code for parent wallets